src/styles/schema-styles.js

```diff
@@ -95,6 +95,22 @@ export default css`
 }
 .toolbar-item:first-of-type { margin:0 2px 0 0;}
 
+/* Expand a block from 0 to something visible, then to probably full screen, then beyond most screens.
```

Comment:
Any thoughts of where we can put this to make it reusable, I don't have a problem duplicating it, but I think we actually want all of these to feel the same, right?

Reply:
Yeah, I managed to reduce to two copies rather than three, but couldn't find anywhere that was shared between all three contexts. That may just be because I don't know the structure of the repo, though.

Reply:
I honestly don't know that there is, maybe it's worth just creating that file, you can find the pattern from the style related imports into these classes

```css
static finalizeStyles() {
    return [
      FontStyles,
      SchemaStyles,
      BorderStyles,
      ...
```

So you could import another one, like `keyframeStyles` or whatever. It's not a big deal, but I think it's worth it.

Reply:
Ah, yes, I'll give that a try.

Reply:
Yep, that seems to work; I've moved the existing "spin" animation in there as well, just seems logical.

Reply:
Awesome.